### PR TITLE
4 section added in css

### LINKED ISSUE #398: Adding Sections in CSS
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

adding this topics with detail,syntax,output and explanation

1. css borders
2. css outline
3. css margin
4. css padding

### Use Case

this will add more docs because urrelt css have only one file
it will add more good docs to be beautiful

### Benefits

will upload only quality and structured content

### Add ScreenShots

_No response_

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Yashgabani845**
## Related Issue

Fixes #398 
## Description

as per i said in issue i have added 
- css padding
- css margin
- css outline
- css border
these sections and one more thing there was no format in CSS before this so i have done in best formar
you can refer it to others if you like it 
i have working video you an refer it

## Type of PR

- [x] Feature enhancement
- [x] Documentation update

## Screenshots / Videos (if applicable)

https://github.com/CodeHarborHub/codeharborhub/assets/127788321/706c2cc1-4089-4b35-b261-7f0b8c9f8dc8

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

as always i tried to maintain best quality but if you want to change anything 
always there for you to improve
and label it whatever you thing about contribute

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

**Comment by Yashgabani845** [comment]
@Ajay-Dhangar 
i have added css sections
i will reqesut you for one thing
please do not assign more than 3/4 issue to 1 person at a time
litrally people are spamming on your repo. it is making trouble toyou and us also,
and solution of 1 leetcode or gfg problem or one docs section  is getting level 2 which is not deserved i have also put soution of kadane but it is not taking much time to make so give it level 1 am also ready if you give my gfg/leetcode solution level

**Comment by Ajay-Dhangar** [comment]
> @Ajay-Dhangar i have added css sections i have small request for you please do not assign more than 3/4 issue to 1 person at a time litrally people are spamming on your repo. it is making trouble toyou and us also, and solution of 1 leetcode or gfg problem or one docs section is getting level 2 which is not deserved i have also put soution of kadane but it is not taking much time to make so give it level 1 i am also ready if you give my gfg/leetcode solution level 1

Thanks for your feedback I will assign level based on work/time.

File: docs/CSS/css-borders.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
+# CSS Borders
+
+```css
+div.all-borders {
+  border: 1px solid black;
+}
+```
+
+#### Output
+
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:10]: **Bad practice**

```md
<div class="all-borders" style={{border: "1px solid black", padding: "10px"}}>
  I have borders on all sides.
</div>
```

**Good Practice**

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       <div class="all-borders" style={{border: "1px solid 
                black", padding: "10px"}}>
                I have borders on all sides.
        </div>   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+</div>
+
+```css
+div.bottom-border {
+  border-bottom: 1px solid red;
+}
+```
+
+#### Output
+
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:22]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+```css
+div.rounded-borders {
+  border: 1px solid black;
+  border-radius: 10px;
+}
+```
+
+#### Output
+
+<div class="rounded-borders" style={{border: "1px solid black", borderRadius: "10px", padding: "10px"}}>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:36]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+</div>
+
+```css
+div.left-border {
+  border-left: 1px solid blue;
+}
+```
+
+#### Output
+
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:47]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+p.inset { border-style: inset; }
+p.outset { border-style: outset; }
+p.none { border-style: none; }
+p.hidden { border-style: hidden; }
+p.mix { border-style: dotted dashed solid double; }
+```
+
+#### Output
+
+<p class="dotted" style={{borderStyle: "dotted"}}>A dotted border.</p>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:70]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+
+p.four {
+  border-style: dotted;
+  border-width: thick;
+}
+```
+
+#### Output
+
+<p class="one" style={{borderStyle: "solid", borderWidth: "5px"}}>5px border-width</p>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:108]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+
+p.three {
+  border-style: dotted;
+  border-color: blue;
+}
+```
+
+#### Output
+
+<p class="one" style={{borderStyle: "solid", borderColor: "red"}}>Red border</p>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:134]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 8]
```diff
@@ -0,0 +1,257 @@
...
+p {
+  border-top-style: dotted;
+  border-right-style: solid;
+  border-bottom-style: dotted;
+  border-left-style: solid;
+}
+```
+
+#### Output
+
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:150]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 9]
```diff
@@ -0,0 +1,257 @@
...
+```css
+p {
+  border-style: dotted solid;
+}
+```
+
+#### Output
+
+<p style={{borderStyle: "dotted solid"}}>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:163]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 10]
```diff
@@ -0,0 +1,257 @@
...
+
+```css
+p {
+  border: 5px solid red;
+}
+```
+
+#### Output
+
+<p style={{border: "5px solid red"}}>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:177]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 11]
```diff
@@ -0,0 +1,257 @@
...
+  Some text
+</p>
+
+```css
+p {
+  border-left: 6px solid red;
+}
+```
+
+#### Output
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:187]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 12]
```diff
@@ -0,0 +1,257 @@
...
+</p>
+
+```css
+p {
+  border-bottom: 6px solid red;
+}
+```
+
+#### Output
+
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:200]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 13]
```diff
@@ -0,0 +1,257 @@
...
+```css
+p {
+  border: 2px solidred;
+  border-radius: 5px;
+}
+```
+
+#### Output
+
+<p style={{border: "2px solid red", borderRadius: "5px"}}>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:216]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 14]
```diff
@@ -0,0 +1,257 @@
...
+```css
+p {
+  border: 2px solid red;
+  border-radius: 15px;
+}
+```
+
+#### Output
+
+<p style={{border: "2px solid red", borderRadius: "15px"}}>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:229]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 15]
```diff
@@ -0,0 +1,257 @@
...
+```css
+p {
+  border: 2px solid red;
+  border-radius: 25px;
+}
+```
+
+#### Output
+
+<p style={{border: "2px solid red", borderRadius: "25px"}}>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:242]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

File: docs/CSS/css-borders.md [hunk 16]
```diff
@@ -0,0 +1,257 @@
...
+```css
+p {
+  border: 2px solid red;
+  border-radius: 50px;
+}
+```
+
+#### Output
+
+<p style={{border: "2px solid red", borderRadius: "50px"}}>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-borders.md:255]: add, for output

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       //your code here   
</BrowserWindow>
```

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
Now, change your all files code based on suggestions

**Comment by Yashgabani845** [comment]
@Ajay-Dhangar done with the changes
it loks very good now thank you for your feedback

**Comment by Yashgabani845** [comment]
@Ajay-Dhangar can you please proceed merge if there is no issue

**Comment by Ajay-Dhangar** [comment]
@Yashgabani845 The site is under maintenance today due to a high volume of test requests. As a result, our queue is currently full. Therefore, I won't be able to merge any contributor PRs today. Once the issue is resolved, I will review and merge all pending PRs. I apologize for the inconvenience and appreciate your patience.

**Pull request merged**